### PR TITLE
[PTX-22947] Print Volume inspect after volume operations in longevity

### PR DESCRIPTION
**Pull request by sayalasomayajula-px**
**What this PR does / why we need it**:
This PR does inspect volumes after certain volume operations tests in longevity

**Which issue(s) this PR fixes** (optional)
Closes # PTX-22947

**Special notes for your reviewer**:
Will attach the logs once  test runs has been completed.
tests covered:
TriggerHAIncrease
TriggerHADecrease
TriggerVolumeClone
TriggerAggrVolDepReplResizeOps
TriggerHAIncreasWithPVCResize
haIncreaseWithErrorInjection
TriggerVolumeResize
TriggerVolumeUpdate
TriggerVolumeIOProfileUpdate
TriggerHAIncreaseAndReboot
TriggerVolumeCreatePXRestart
TriggerHAIncreaseAndCrashPX
TriggerHAIncreaseAndPXRestart

log:https://jenkins.pwx.dev.purestorage.com/job/Users/job/krishna/job/tp-pxe-byoc/2065/consoleFull
Dashboard url: https://aetos.pwx.purestorage.com/resultSet/testSetID/590718

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -798,7 +802,7 @@ func ValidatePDB(pdbValue int, allowedDisruptions int, initialNumNodes int, isCl
 		}
 		if initialNumNodes-currentNumNodes > 1 {
 			*isClusterParallelyUpgraded = true
-			
+
```
Comment (spai-px) [inline tests/common.go:805]: Small nit pick, seems like you have added a space here. Please remove it.

Comment (sayalasomayajula-px) [reply tests/common.go]: removed the whitespace

File: tests/testTriggers.go [hunk 2]
```diff
@@ -2750,6 +2758,8 @@ func TriggerVolumeClone(contexts *[]*scheduler.Context, recordChan *chan *EventR
 					log.InfoD(stepLog)
 					log.Infof("Calling CloneVolume()...")
 					clonedVolID, err = Inst().V.CloneVolume(vol.ID)
+					log.InfoD("Printing the volume inspect for the volume:%s ,volID:%s and namespace:%s", vol.Name, vol.ID, vol.Namespace)
```
Comment (lsrinivas-pure) [inline tests/testTriggers.go:2761]: remove the log.infoD or change it to Debugf
Also print the volume details after source node event also

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]

File: tests/testTriggers.go (hunk 2 above)
Comment (sayalasomayajula-px) [reply tests/testTriggers.go]: changed it from infod to debugf and also added the volumeinspect after sourcenode event

**Comment by lsrinivas-pure** [comment]
Add the logs for 
TriggerVolumeResize
TriggerVolumeUpdate
TriggerVolumeIOProfileUpdate

File: tests/testTriggers.go [hunk 3]
```diff
@@ -1066,11 +1066,15 @@ func haIncreaseWithErrorInjection(event *EventRecord, contexts *[]*scheduler.Con
...
 					}
 				}
+				log.Debugf("Printing the volume inspect for the volume:%s ,volID:%s and namespace:%s", v.Name, v.ID, v.Namespace)
+				PrintInspectVolume(v.ID)
 
 				if err == nil {
 					HaIncreaseRebootTargetNode(event, selctx, v, storageNodeMap, errorInj)
 					HaIncreaseRebootSourceNode(event, selctx, v, storageNodeMap, errorInj)
 				}
+				log.Debugf("Printing the volume inspect for the volume:%s ,volID:%s and namespace:%s after HA increase and reboot of the source node ", v.Name, v.ID, v.Namespace)
```
Comment (lsrinivas-pure) [inline tests/testTriggers.go:1076]: add this after operation on target and also after source node

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/testTriggers.go (hunk 3 above)
Comment (sayalasomayajula-px) [reply tests/testTriggers.go:1076]: added

**Review by spai-px — APPROVED** [review]
LGTM

File: tests/common.go [hunk 4]
```diff
@@ -5683,6 +5687,8 @@ func HaIncreaseErrorInjectionTargetNode(event *EventRecord, ctx *scheduler.Conte
 
 			if err != nil {
 				err = fmt.Errorf("error getting replication factor for volume %s, Error: %v", v.Name, err)
+				log.Error(err)
+				UpdateOutcome(event, err)
```
Comment (lsrinivas-pure) [inline tests/common.go:5691]: remove these changes here

File: tests/common.go [hunk 5]
```diff
@@ -6040,6 +6046,8 @@ func HaIncreaseErrorInjectSourceNode(event *EventRecord, ctx *scheduler.Context,
 					})
 			} else {
 				err = fmt.Errorf("error getting current replication factor for volume %s, Error: %v", v.Name, err)
+				log.Error(err)
+				UpdateOutcome(event, err)
```
Comment (lsrinivas-pure) [inline tests/common.go:6050]: remove these changes here

File: tests/common.go (hunk 4 above)
Comment (sayalasomayajula-px) [reply tests/common.go]: removed

File: tests/common.go (hunk 5 above)
Comment (sayalasomayajula-px) [reply tests/common.go]: removed

File: tests/common.go [hunk 6]
```diff
@@ -7786,6 +7790,8 @@ func CreateMultiVolumesAndAttach(wg *sync.WaitGroup, count int, nodeName string)
 		createdVolIDs[volId] = volPath
 		log.Infof("Volume %s attached to path %s", volId, volPath)
 		count--
+		log.InfoD("Printing the volume inspect for the volume")
```
Comment (kphalgun-px) [inline tests/common.go:7793]: How about log.Debugf and include volume details as well?

Comment (sayalasomayajula-px) [reply tests/common.go]: changed it, thanks for notifying

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**